archive/archive_api/src/conftest.py

```diff
@@ -60,3 +76,16 @@ def create_table(dynamodb_client, table_name):
         dynamodb_client.get_waiter('table_exists').wait(TableName=table_name)
     except dynamodb_client.exceptions.ResourceInUseException:
         pass
+
+
+@pytest.fixture
+def bag_id():
+    return str(uuid.uuid4())
+
+
+@pytest.fixture
+def bucket_bag(s3_client):
+    bucket_name = 'test-python-bucket-bag'
```

Comment:
Does this bucket name need a random suffix so we can run tests in parallel?

Reply:
If so then you probably want to update `bucket` in shared_conftest.py too: 
```@pytest.fixture
def bucket(s3_client):
    bucket_name = 'test-python-bucket'
    s3_client.create_bucket(Bucket=bucket_name)
    yield bucket_name```